src/styles/index.css

```diff
@@ -0,0 +1,72 @@
+@keyframes watch-animation-rotate {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+body {
+  margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  color: #000;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 1vmin;
```

Comment:
The border property does not have a color. You should add a color to it. For example: `border: 1vmin dotted #000;`